packages/cli/src/databases/mysqldb/migrations/1636626154933-CreateUserManagement.ts

```diff
@@ -0,0 +1,158 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { v4 as uuid } from 'uuid';
+import config = require('../../../../config');
+import { loadSurveyFromDisk } from '../../utils/migrationHelpers';
+
+export class CreateUserManagement1636626154933 implements MigrationInterface {
+	name = 'CreateUserManagement1636626154932';
+
+	public async up(queryRunner: QueryRunner): Promise<void> {
+		const tablePrefix = config.get('database.tablePrefix');
+
+		await queryRunner.query(
+			'CREATE TABLE `' + tablePrefix + 'role` ( ' +
+				'`id` int NOT NULL AUTO_INCREMENT, ' +
+				'`name` varchar(32) NOT NULL, ' +
+				'`scope` varchar(250) NOT NULL, ' +
+				'`createdAt` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP, ' +
+				'`updatedAt` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP, ' +
+				'PRIMARY KEY (`id`), ' +
+				'UNIQUE KEY `UQ_5b49d0f504f7ef31045a1fb2eb8` (`scope`,`name`) ' +
+			');'
+		);
+
+		await queryRunner.query(
+			'CREATE TABLE `' + tablePrefix + 'user` ( ' +
+				'`id` VARCHAR(100) NOT NULL, ' +
+				'`email` VARCHAR(254) NULL DEFAULT NULL, ' +
+				'`firstName` VARCHAR(32) NULL DEFAULT NULL, ' +
+				'`lastName` VARCHAR(32) NULL DEFAULT NULL, ' +
+				'`password` VARCHAR(200) NULL DEFAULT NULL, ' +
+				'`resetPasswordToken` VARCHAR(200) NULL DEFAULT NULL, ' +
```

Comment:
why the 200 length here? if we want to use the full byte we can also go for 254/255 (for no index)

for `resetPasswordToken` we use currently a UUID so we could even fix the length here, for future proofing I would probably keep it variable

also is 200 enough for the password hash?